learn_django/trip_project/models.py

```diff
@@ -8,9 +8,11 @@ class Host(models.Model):
     host_number = models.CharField(max_length=20)
 
 
+class Location(models.Model):
+    LOCATION_CHOICES = [('NR', 'Naran'), ('HN', 'Hunza'), ('SK', 'Skardu'), ('LHE', 'Lahore'), ('ISB', 'Islamabad')]
+    location_name = models.CharField(max_length=100, choices=LOCATION_CHOICES)
```

Comment:
you dont' need `choices` here, just a Charfield would work. We shouldn't limit the choices.